runtime/src/main/java/org/corfudb/runtime/view/metadata/Checkpoint.java

```diff
@@ -0,0 +1,14 @@
+package org.corfudb.runtime.view.metadata;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * This class defines metadata associated with a checkpoint
+ */
+public class Checkpoint {
+    private UUID checkpointStream;
+    private long startAddress;
+    private long endAddress;
+    List<Long> checkpointRecords;
```

Comment:
I don't see a need right away. however, checkpointWriter does keep track of those. So it might not be a bad idea to retain them for any potential future use. Hopefully, with reasonable batch sizes, this list will not be very long.